public/locales/uk/common.json

```diff
@@ -232,5 +234,10 @@
     "android_soon": "Версія для Android незабаром!",
     "play": "відтворити",
     "new_game": "Нова гра"
+  },
+  "secondary_navigation": {
```

Comment:
@met  What is the reason behind splitting this into 3 different pages? It's not that much content, splitting it up increases navigation complexity and decreases the chance that people actually see all of it or find what they are looking for. 

We are turning 1 click into 6 clicks. I would be lazy to click that many times even if I cared about the "About" page, which most people don't :D

Reply:
@JuroUhlar It was a little but invention of @mikekubn . 😄 My first idea was to have team members on a speacial page and rest on the about page. But I am completelly OK with just one page as it will do not need any more navigation.

Reply:
I don't mind. I think that it will not good choice leave all section in one page but I told, I do not care. 🤷🏼‍♂️